.sauce/sentry-uitest-android-benchmark.yml

```diff
@@ -2,25 +2,59 @@ apiVersion: v1alpha
 kind: espresso
 sauce:
   region: us-west-1
-  # Controls how many suites are executed at the same time (sauce test env only).
-  concurrency: 1
+  concurrency: 2
   metadata:
     name: Android benchmarks with Espresso
     tags:
       - benchmarks
       - android
+
+defaults:
+  timeout: 30m
+
 espresso:
   app: ./sentry-android-integration-tests/sentry-uitest-android-benchmark/build/outputs/apk/release/sentry-uitest-android-benchmark-release.apk
   testApp: ./sentry-android-integration-tests/sentry-uitest-android-benchmark/build/outputs/apk/androidTest/release/sentry-uitest-android-benchmark-release-androidTest.apk
+
 suites:
-  name: "Android Benchmarks"
-  devices:
-    - name: "Google Pixel 2"
-      platformVersion: 11
-    - id: Google_Pixel_2_real_us
-  testOptions:
-    useTestOrchestrator: true
-# Controls what artifacts to fetch when the suite on Sauce Cloud has finished.
+
+  # Devices are chosen so that there is a high-end and a low-end device for each api level
+  - name: "Android 12 (api 31)"
+    devices:
+      - id: Google_Pixel_6_Pro_real_us # Google Pixel 6 Pro - api 31 (12)
+      - id: Google_Pixel_3_12_real_us # Google Pixel 3 - api 31 (12)
+    testOptions:
+      useTestOrchestrator: true
+
+  - name: "Android 11 (api 30)"
+    devices:
+      - id: OnePlus_9_Pro_real_us # OnePlus 9 Pro - api 30 (11)
+      - id: Google_Pixel_2_real_us # Google Pixel 2 - api 30 (11)
+    testOptions:
+      useTestOrchestrator: true
+
+# Commenting for the moment, due to the error "Cannot install test-services-1.4.1.apk on device" on low Android versions
```

Comment:
Why do we get this error? Should we fix or remove the comments?

Reply:
The errors are related to an issue with Android test orchestrator on old versions.
I will fix the issue, but in another pr, so we can understand how the sauce labs runs work with the other commits (adding devices would make the tests take longer)